src/services/fileServices/MdPageServices/PageService.ts

```diff
@@ -44,13 +44,13 @@ const FRONTEND_URL = process.env.FRONTEND_URL || "http://localhost:3000"
 // This is (at a glance), either a failure to read from github
 // or the parsing fails (we have a sentinel value of an empty string)
 const withErrorHandler = (promise: Promise<PageInfo>) =>
-  ResultAsync.fromPromise(promise, () => new BaseIsomerError()).andThen(
+  ResultAsync.fromPromise(promise, () => new BaseIsomerError({})).andThen(
     (data) =>
       // NOTE: This is a fail-safe check as `yaml.parse`
       // doesn't guarantee existence.
       data?.content?.frontMatter?.permalink
         ? ok(data)
-        : err(new BaseIsomerError())
+        : err(new BaseIsomerError({}))
```

Comment:
actually maybe we want to set the initial argument as `Object | undefined` if we have defaults for them/they are safe to omit

Reply:
ideally would prefer this to be explicit as this is a new format introduced